### PR TITLE
docs/readme

### PR DESCRIPTION
**Pull request by azdanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
...
 </p>
 
-## Features
+<h2>Features</h2>
```
Comment (jorgegonzalez) [inline README.md]: Why this change?

Comment (azdanov) [reply README.md]: I use a plugin in VSCode to update TOC. When html is used directly it ignores those headings.

Easier to update for those who might use this plugin by helping prevent accidental updates.

https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
